Jenkinsfile.dcos-commons-base

```diff
@@ -0,0 +1,42 @@
+pipeline {
+    agent {
+        label 'mesos'
+    }
+ 
+    environment {
+        IMAGE = "dcos-commons-base"
+        DOCKERFILE = "Dockerfile.base"
+ 
+        // Get credentials for publishing to Docker hub.
+        DOCKER = credentials('docker-hub-credentials')
```

Comment:
how are these generated / where are these stored (where can someone can see these in plain text) ?